resources/views/includes/navbar.blade.php

```diff
@@ -14,8 +14,8 @@
                     <div class="nav-item dropdown">
                         <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown">Jobs</a>
                         <div class="dropdown-menu rounded-0 m-0">
-                            <a href="job-list.html" class="dropdown-item">Job List</a>
-                            <a href="job-detail.html" class="dropdown-item">Job Detail</a>
+                            <a href="{{route('job-list')}}" class="dropdown-item">Job List</a>
+                            <a href="{{route('detail')}}" class="dropdown-item">Job Detail</a>
                         </div>
                     </div>
                     <div class="nav-item dropdown">
```

Comment:
add this script at the end of navbar.blade.php
```js
<script>
    anchors = Array.from(document.getElementsByClassName("nav-item nav-link"))

    anchors.forEach(function (anchor) {
        if (anchor.href === window.location.href) {
            anchor.className = "nav-item nav-link active"
        } else {
            anchor.className = "nav-item nav-link"
        }
    })

</script>
```

Reply:
it will help with active links

Reply:
hmmm I will think about it for now just use the script

Reply:
i found a good function in laravel specifically in blade part 
Simplify the Blade Logic: Use the built-in request() helper in Blade to check if the current route or URL matches the active link. ex:
<a href="{{ route('index') }}" class="nav-item nav-link {{ request()->routeIs('index') ? 'active' : '' }}">Home</a>
and I updated all links and now works successfully active

Reply:
nice

Reply:
I have a little exercise for U if you want, can you make a custom directive so i can call @active('routeName') and it prints active, this will be more cleaner 

@Omneya-Abdelmajeed  @Basma-Abdallah 
try to make it

Reply:
Will try (إن شاء الله) 😃